addons/l10n_uy/data/template/account.account-uy.csv

```diff
@@ -71,8 +72,9 @@
 "uy_code_21320","Other debts","21320","liability_current","False","Otras deudas"
 "uy_code_21401","Minimum Sales Tax","21401","liability_current","False","Iva Ventas Mínima"
 "uy_code_21402","Basic Sales Tax","21402","liability_current","False","Iva Ventas Básica"
-"uy_code_21403","Tax to be Paid","21403","liability_payable","True","Iva a Pagar"
-"uy_code_21404","Withholding Tax","21404","liability_current","False","Iva Retenido"
+"uy_code_21403","Exempt Sales Tax","21403","liability_current","False","Iva Ventas Exento"
+"uy_code_21404","Tax to be Paid","21404","liability_payable","True","Iva a Pagar"
+"uy_code_21405","VAT Withholding","21405","liability_current","False","Iva Retenido"
```

Comment:
@JulienVR I think you're right, the chart template reload only updates account tags on existing accounts, not account type. 
https://github.com/odoo/odoo/blob/8acd7aae3ec375ef1d5a0e8d821bbd300d151ffd/addons/account/models/chart_template.py#L369-L373

@zaoral Since you seem to want to create a new account here (`IVA Ventas Exento`), you could just give it a code and xmlid which aren't yet taken in the CoA.

Reply:
@JulienVR @antoine162 
Thank you for the review and all the info
I have updated the code with your suggestion
Is ready for your review

NOTE: Also, I need to ask, what will happen on module migration if the xml_id changed between versions?

Best Regards

Reply:
@zaoral All the `data/template` data, including the accounts, aren't 'data' in the usual sense in that they aren't loaded with the module. Instead, they are managed by the chart template loading (when you select a localization) or reloading (when you click on the 'reload' button). 

Module migration doesn't trigger chart template reloading, so it doesn't do anything to the accounts.

Reply:
Thank you for the info @antoine162